src/Symfony/Component/Notifier/Bridge/GatewayApi/README.md

```diff
@@ -16,6 +16,31 @@ where:
 
 See your account info at https://gatewayapi.com
 
+Adding options to a Message
+---------------------------
+
+With a GatewayApi Message, you can use the `GatewayApiOptions` class to add message options.
+
+```php
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Bridge\GatewayApi\GatewayApiOptions;
+
+$smsMessage = new SmsMessage('+1411111111', 'My message');
+
+// Create GatewayApiOptions options
+$gatewayApiOptions = (new GatewayApiOptions())
+    ->class(123)
```

Comment:
I think there is a bug here, in [documentation](https://gatewayapi.com/docs/apis/rest/), type of `class` is a string.
Are you ok if I create a PR to fix it ?

/cc @gnito-org

Reply:
from https://gatewayapi.com/docs/apis/rest/, this is a string, please update yes